packages/electron-builder-squirrel-windows/src/SquirrelWindowsTarget.ts

```diff
@@ -102,22 +101,21 @@ export default class SquirrelWindowsTarget extends Target {
     const appName = this.appName
     const options: SquirrelOptions = {
       name: appName,
-      productName: this.options.name || appInfo.productName,
       appId: this.options.useAppIdAsId ? appInfo.id : appName,
       version: appInfo.version,
       description: appInfo.description,
-      // better to explicitly set to empty string, to avoid any nugget errors
+      exe: `${this.packager.platformSpecificBuildOptions.executableName || this.options.name || appInfo.productName}.exe`,
       authors: appInfo.companyName || "",
       iconUrl,
       extraMetadataSpecs: projectUrl == null ? null : `\n    <projectUrl>${projectUrl}</projectUrl>`,
       copyright: appInfo.copyright,
       packageCompressionLevel: parseInt((process.env.ELECTRON_BUILDER_COMPRESSION_LEVEL || packager.compression === "store" ? 0 : 9) as any, 10),
-      vendorPath: await getBinFromUrl("Squirrel.Windows", "1.9.0", "zJHk4CMATM7jHJ2ojRH1n3LkOnaIezDk5FAzJmlSEQSiEdRuB4GGLCegLDtsRCakfHIVfKh3ysJHLjynPkXwhQ=="),
```

Comment:
If we're removing this, we need to validate the validity/integrity of the new package. To replicate the previoius functionality, electron-builder should validate the hash value of the exe's in the npm package by hardcoding the hashes in our repo. This will replicate what the current `getBinFromUrl` is doing with the hardcoded hash

Reply:
This is a bit difficult. In the [https://github.com/electron/windows-installer](https://github.com/electron/windows-installer) library, the vendor(https://github.com/electron/windows-installer/tree/main/vendor) is already included, and it contains all the exe files. We no longer need to download them separately as they are included with the package download. So, it seems we don't need to validate them again, right?

Reply:
We need to validate them from our side for security reasons. An npm package can change hashes for the same version with a force push IIRC. That's what electron-builder-binaries is used for.

An alternative route is to move the vendor files into this monorepo as separate github release artifacts (but not published to npm). 

The previous logic only downloads the necessary vendor packages when they are necessary. We shouldn't be downloading all resources on `npm install`